docs/openapi/components/schemas/common/IntellectualPropertyRights.yml

```diff
@@ -52,7 +52,7 @@ example: |-
     "intellectualPropertyRightsType": "Copyright",
     "intellectualPropertyRightsProduct": {
       "type": ["Product"],
-      "id": "ecommerce-marketplace.example.com/catalog/lawn-flamingos-357fc1f4",
+      "id": "https://ecommerce-marketplace.example.com/catalog/lawn-flamingos-357fc1f4",
```

Comment:
surprisingly this value can be mutable when it is not an IRI ; )

We really need to just sign the bytes...

Reply:
Ohhh, rookie-mistake!! Shame-shame-shame...